lib/object.js

```diff
@@ -128,6 +128,27 @@ newObject.filterOut = function(callback)
 };
 
 /**
+ * For each property in the mapping object, if 'this' has a property of the same name, then the name of the property in 'this'
+ * is renamed to the value corresponding to the property from the mapping object.
+ * Example:
+ * var bla = {'yo': 'hi', 'bo': 'hi'};
+ * bla.renameProperties({'bo': 'do'});
+ * THEN 
+ * bla.hasOwnProperty('do') === true
+ * bla.do === 'hi'
+ */
+newObject.renameProperties = function(mappingObject)
```

Comment:
I always ask myself what would happen if the original object already has a method `renameProperties()`. Improbable as it may seem, across many users it is almost certain to come up eventually.

You realize that if the prototype has such a function it is not rewritten, right?